widgetry/src/lib.rs

```diff
@@ -41,13 +41,14 @@ pub use crate::geom::{GeomBatch, RewriteColor};
 pub use crate::input::UserInput;
 pub use crate::runner::{run, Settings};
 pub use crate::screen_geom::{ScreenDims, ScreenPt, ScreenRectangle};
-pub use crate::style::Style;
-pub use crate::text::{Line, Text, TextExt, TextSpan};
+pub use crate::style::{buttons::StyledButtons, Style};
+pub use crate::text::{Font, Line, Text, TextExt, TextSpan};
 pub use crate::tools::warper::Warper;
 pub use crate::tools::Cached;
 pub use crate::widgets::autocomplete::Autocomplete;
 pub(crate) use crate::widgets::button::Button;
-pub use crate::widgets::button::{Btn, MultiButton};
+// REVIEW: move ContentMode to upper level?
```

Comment:
Is this comment out-of-date? Defining it in this file seems reasonable to me; it's only used in `widgets/button.rs` right now, but I'd guess it might come in handy elsewhere.

Reply:
Yes, it was out of date. Deleted!